scenarios/result_export/export_result_tableau/export_result_tablueau.dig

```diff
@@ -13,7 +13,19 @@ _export:
   content_type: application/json
   content_format: json
   content:
-    '{"result":"{\"type\":\"tableau\",\"host\":\"${secret:tableauHost}\",\"username\":\"${secret:tableauUser}\",\"password\":\"${secret:tableauPass}\",\"ssl\":\"true\",\"server_version\":\"online\",\"datasource\":\"${secret:tableauDatasource}\",\"site\":\"${secret:tableauSite}\",\"project\":\"${secret:tableauProject}\",\"mode\":\"replace\"}"}'
+    result: |
```

Comment:
@kamikaseda BTW, this filename is typo. shall I fix filename?

Reply:
Ah, yes please! thank you.

Reply:
fixed! https://github.com/treasure-data/treasure-boxes/pull/269/commits/cc3d9b44d1e9f74587a28aa87458315c8184171a